### PR TITLE
feat: Tracing span for deploy action

### PR DESCRIPTION
**Pull request by jakmeier**

### CODE REVIEW

File: runtime/runtime/src/actions.rs [hunk 1]
```diff
@@ -436,6 +436,7 @@ pub(crate) fn action_deploy_contract(
     apply_state: &ApplyState,
     current_protocol_version: ProtocolVersion,
 ) -> Result<(), StorageError> {
+    let _span = tracing::debug_span!(target: "vm", "action_deploy_contract").entered();
```
Comment (matklad) [inline runtime/runtime/src/actions.rs:439]:
```suggestion
    let _span = tracing::debug_span!(target: "runtime", "action_deploy_contract").entered();
```

this is still runtime

File: runtime/runtime-params-estimator/src/main.rs [hunk 2]
```diff
@@ -71,6 +71,8 @@ struct CliArgs {
     /// Print extra debug information
     #[clap(long, multiple(true), possible_values=&["io", "rocksdb", "least-squares"])]
     debug: Vec<String>,
+    #[clap(long)]
+    tracing_span_tree: bool,
```
Comment (matklad) [inline runtime/runtime-params-estimator/src/main.rs:75]: :facepalm: it literally didn't occur to me that we can make this a command-line argument.

**Review by matklad — APPROVED** [review]
LGTM with vm changed to runtime in one case!

File: runtime/runtime-params-estimator/src/main.rs (hunk 2 above)
Comment (matklad) [reply runtime/runtime-params-estimator/src/main.rs:75]: Could you add a doc comment here though? So that it gets to the `--help` message

**Review by bowenwang1996 — APPROVED** [review]

**Pull request merged**